cluster-operator/src/test/java/io/strimzi/operator/cluster/model/CruiseControlTest.java

```diff
@@ -1085,18 +1088,35 @@ public void testMetricsParsingFromConfigMap() {
                 .build();
         CruiseControl cc = createCruiseControl(kafka, NODES, STORAGE, Map.of());
 
-        assertThat(cc.metrics().isEnabled(), is(true));
-        assertThat(cc.metrics().getConfigMapName(), is("my-metrics-configuration"));
-        assertThat(cc.metrics().getConfigMapKey(), is("config.yaml"));
+        assertThat(cc.metrics(), is(notNullValue()));
```

Comment:
I think you can just do `assertThat(cc.metrics(), notNullValue());`. That said, it seems Strimzi uses `is(notNullValue())` all over its code base, so let's keep it as it.